x/bounty/types/program.go

```diff
@@ -34,7 +38,8 @@ func (m *Finding) ValidateBasic() error {
 		m.SeverityLevel < 0 ||
 		int(m.SeverityLevel) >= len(SeverityLevel_name) ||
 		int(m.FindingStatus) >= len(FindingStatus_name) {
-		return fmt.Errorf("finding data is error:%s", m.String())
+		return fmt.Errorf("finding programId[%d] or findingId[%d] or SeverityLevel[%d] or findingStatus[%d] is error",
+			m.ProgramId, m.FindingId, m.SeverityLevel, m.FindingStatus)
```

Comment:
grammar
should use switch and if-else to actually find out what the error is rather than bunching it up on one condition